### PR TITLE
Update Hausa locale information

### PR DESCRIPTION
**Pull request by petyaraykovska**
Request: https://make.wordpress.org/polyglots/2016/04/22/request-for-adding-hausa-locale/

### CODE REVIEW

File: locales/locales.php [hunk 1]
```diff
@@ -961,15 +961,15 @@ public function __construct() {
...
-		$ha->text_direction = 'rtl';
-		$ha->google_code = 'ha';
-		$ha->facebook_locale = 'ha_NG';
+		$hau = new GP_Locale();
+		$hau->english_name = 'Hausa';
+		$hau->native_name = 'Harshen Hausa';
+		$hau->lang_code_iso_639_1 = 'ha';
+		$hau->lang_code_iso_639_2 = 'hau';
+		$hau->lang_code_iso_639_3 = 'hau';
+		$hau->slug = 'hau';
```
Comment (toolstack) [inline locales/locales.php]: Any particular reason for the slug change?

File: locales/locales.php [hunk 2]
```diff
@@ -961,15 +961,15 @@ public function __construct() {
...
-		$ha->native_name = 'هَوُسَ';
-		$ha->lang_code_iso_639_1 = 'ha';
-		$ha->lang_code_iso_639_2 = 'hau';
-		$ha->slug = 'ha';
-		$ha->text_direction = 'rtl';
-		$ha->google_code = 'ha';
-		$ha->facebook_locale = 'ha_NG';
+		$hau = new GP_Locale();
+		$hau->english_name = 'Hausa';
+		$hau->native_name = 'Harshen Hausa';
```
Comment (toolstack) [inline locales/locales.php]: The native name should not be in english, this should be in the locale language.

Comment (petyaraykovska) [inline locales/locales.php]: Languages are evolving. The locale was originally added in Arabic script, including text direction. Both script and text direction are changed in this version at the request of the person who will maintain the locale. The native name is not in English, it's in boko, which is a latin alphabeth used for Hausa. https://en.wikipedia.org/wiki/Boko_alphabet

File: locales/locales.php (hunk 1 above)
Comment (petyaraykovska) [inline locales/locales.php]: We're using ISO 639-3 codes for new WordPress locales and where possible adapting old ones as well (https://make.wordpress.org/polyglots/handbook/translating/requesting-a-new-locale/#find-your-language-code). Hausa has existed in GlotPress for a while but it doesn't have a WP locale and didn't have a site. The ISO 639-3 code is applied to the slugs as well as the language code for consistency. The site is already created with the slug `hau`.

**Comment by ocean90** [comment]
@petyaraykovska Instead of updating the existing locale, let's add `hau` as a new one.

File: locales/locales.php [hunk 3]
```diff
@@ -962,14 +962,14 @@ public function __construct() {
...
-		$ha->google_code = 'ha';
-		$ha->facebook_locale = 'ha_NG';
+ -		$ha->english_name = 'Hausa';
+ -		$ha->native_name = 'هَوُسَ';
+ -		$ha->lang_code_iso_639_1 = 'ha';
+ -		$ha->lang_code_iso_639_2 = 'hau';
+ -		$ha->slug = 'ha';
+ -		$ha->text_direction = 'rtl';
+ -		$ha->google_code = 'ha';
+ -		$ha->facebook_locale = 'ha_NG';
```
Comment (ocean90) [inline locales/locales.php:972]: The `-` shouldn't be there...

**Comment by ocean90** [comment]
Please create one PR per locale.

**Comment by ocean90** [comment]
New PRs: #549 and #550

**Pull request closed without merging**